src/WrapperApp/components/Results/Results3D.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Button, Stack, Typography } from '@mui/material';
+import { Page3D } from '../../../JsRoot/GraphData';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+
+export function Result3D(props: { page: Page3D; title?: string }) {
+	const { page, title } = props;
+
+	return (
+		<Box sx={{ margin: '1rem 0' }}>
```

Comment:
use spacing value from theme to define margin
```suggestion
		<Box sx={{ margin: ({spacing}) => spacing(1, 0) }}>
```